File: sub-mysql-web/src/main/java/org/github/base/mapper/IMemberMapper.java
package org.github.base.mapper;

import org.github.base.entity.MemberEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * <p>
 *  Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-20
 */
@MyBatisMapper
public interface IMemberMapper extends IMapper<MemberEntity> {}

File: sub-mysql-course-service/src/main/java/org/github/mysql/course/base/mapper/IMemberMapper.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.course.base.mapper;

import org.github.mysql.course.base.entity.MemberEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * 会员 Mapper 接口
 *
 * @author JYD_XL
 */
@MyBatisMapper
interface IMemberMapper : IMapper<MemberEntity>
